### ISSUE TITLE
Exception: Can't load log handler "java.util.logging.FileHandler"

### ISSUE THREAD

**Issue by chaminda** [ballerinalang/ballerina#2036]
Following message noticed when running balerina samples

```
Can't load log handler "java.util.logging.FileHandler"
java.nio.file.NoSuchFileException: logs/carbon.log.lck
java.nio.file.NoSuchFileException: logs/carbon.log.lck
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newFileChannel(UnixFileSystemProvider.java:177)
	at java.nio.channels.FileChannel.open(FileChannel.java:287)
	at java.nio.channels.FileChannel.open(FileChannel.java:335)
	at java.util.logging.FileHandler.openFiles(FileHandler.java:459)
	at java.util.logging.FileHandler.<init>(FileHandler.java:263)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at java.util.logging.LogManager$5.run(LogManager.java:966)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.util.logging.LogManager.loadLoggerHandlers(LogManager.java:958)
	at java.util.logging.LogManager.initializeGlobalHandlers(LogManager.java:1578)
	at java.util.logging.LogManager.access$1500(LogManager.java:145)
	at java.util.logging.LogManager$RootLogger.accessCheckedHandlers(LogManager.java:1667)
	at java.util.logging.Logger.getHandlers(Logger.java:1777)
	at java.util.logging.Logger.log(Logger.java:735)
	at java.util.logging.Logger.doLog(Logger.java:765)
	at java.util.logging.Logger.logp(Logger.java:931)
	at org.apache.commons.logging.impl.Jdk14Logger.log(Jdk14Logger.java:87)
	at org.apache.commons.logging.impl.Jdk14Logger.warn(Jdk14Logger.java:260)
	at org.apache.axiom.util.stax.dialect.StAXDialectDetector.detectDialect(StAXDialectDetector.java:214)
	at org.apache.axiom.util.stax.dialect.StAXDialectDetector.getDialect(StAXDialectDetector.java:199)
	at org.apache.axiom.util.stax.dialect.StAXDialectDetector.getDialect(StAXDialectDetector.java:163)
	at org.apache.axiom.util.stax.dialect.StAXDialectDetector.getDialect(StAXDialectDetector.java:177)
	at org.apache.axiom.om.util.StAXUtils$1.run(StAXUtils.java:447)
	at org.apache.axiom.om.util.StAXUtils$1.run(StAXUtils.java:425)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.axiom.om.util.StAXUtils.newXMLInputFactory(StAXUtils.java:425)
	at org.apache.axiom.om.util.StAXUtils.getXMLInputFactory_perClassLoader(StAXUtils.java:495)
	at org.apache.axiom.om.util.StAXUtils.getXMLInputFactory(StAXUtils.java:168)
	at org.apache.axiom.om.util.StAXUtils.getXMLInputFactory(StAXUtils.java:133)
	at org.apache.axiom.om.util.StAXUtils.createXMLStreamReader(StAXUtils.java:234)
	at org.apache.axiom.om.impl.common.factory.AbstractOMMetaFactory.createXMLStreamReader(AbstractOMMetaFactory.java:126)
	at org.apache.axiom.om.impl.common.factory.AbstractOMMetaFactory.createOMBuilder(AbstractOMMetaFactory.java:167)
	at org.apache.axiom.om.OMXMLBuilderFactory.createOMBuilder(OMXMLBuilderFactory.java:290)
	at org.apache.axiom.om.OMXMLBuilderFactory.createOMBuilder(OMXMLBuilderFactory.java:274)
	at org.apache.axiom.om.util.AXIOMUtil.stringToOM(AXIOMUtil.java:54)
	at org.apache.axiom.om.util.AXIOMUtil.stringToOM(AXIOMUtil.java:39)
	at org.ballerinalang.model.values.BXML.<init>(BXML.java:46)
	at org.ballerinalang.bre.BLangExecutor.visit(BLangExecutor.java:857)
	at org.ballerinalang.model.expressions.BacktickExpr.execute(BacktickExpr.java:53)
	at org.ballerinalang.bre.BLangExecutor.visit(BLangExecutor.java:157)
	at org.ballerinalang.model.statements.VariableDefStmt.execute(VariableDefStmt.java:66)
	at org.ballerinalang.bre.BLangExecutor.visit(BLangExecutor.java:138)
	at org.ballerinalang.model.statements.BlockStmt.execute(BlockStmt.java:64)
	at org.ballerinalang.bre.BLangExecutor.visit(BLangExecutor.java:605)
	at org.ballerinalang.model.expressions.FunctionInvocationExpr.executeMultiReturn(FunctionInvocationExpr.java:106)
	at org.ballerinalang.bre.BLangExecutor.visit(BLangExecutor.java:289)
	at org.ballerinalang.model.statements.FunctionInvocationStmt.execute(FunctionInvocationStmt.java:51)
	at org.ballerinalang.bre.BLangExecutor.visit(BLangExecutor.java:138)
	at org.ballerinalang.model.statements.BlockStmt.execute(BlockStmt.java:64)
	at org.ballerinalang.BLangProgramRunner.runMain(BLangProgramRunner.java:126)
	at org.ballerinalang.launcher.BProgramRunner.runMain(BProgramRunner.java:50)
	at org.ballerinalang.launcher.Main$RunMainCmd.execute(Main.java:364)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:296)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:50)
[2017-02-20 9:51:01,608]  WARNING {org.apache.axiom.util.stax.dialect.StAXDialectDetector detectDialect} - Unable to determine dialect of the StAX implementation at jar:file:/home/chaminda/Documents/Testing/ballerina/dist/tool/ballerina-tools-0.8.0-SNAPSHOT/bre/lib/swagger-ballerina-generator-0.8.0.jar!/  
```

**Comment by chanakaudaya** [comment]
duplicate of #2042 #2038

**Comment by chanakaudaya** [comment]
This is fixed with 0.8.0 GA.

**Comment by Viyond** [comment]
how to fix, can u show the code change?
